.github/pull_request_template.md

```diff
@@ -1,62 +1,9 @@
 <!-- Specify the issue ID that this pull request is solving -->
-Fix #XXX
+Fixes #XXX
+<!-- Make sure that the issue provides steps to reproduce it -->
 
 <!-- Describe the changes introduced by this pull request -->
 
 <!--
-  Please provide a unit or e2e test if possible.
-
-  To simplify the reviewing process of this pull request,
-  please explain how it should be tested.
-  The following is just an example
-
-## Test
-
-To test this pull request, you can run the following commands:
-
-```shell
-cd <to_package_directory>
-go test
-```
-
--->
-
-## Additional Information
-
-### Trade-off
-
-<!-- Please describe, if any, the trade-offs that you found acceptable in this pull request -->
-
-### Potential improvement
-
-<!-- Please describe, if any, potential improvement that you are envisioning -->
-
-<!-- Please Uncomment following block if looking for QA validation
-
-## Additional QA
```

Comment:
Shouldn't we _move_ the QA template somewhere else, instead of deleting it? Could we have it eg. in `.github/ISSUE_TEMPLATE`, or somewhere we could use it as reference when needed?

Reply:
> could prevent users from creating issues because of the huge amount of content required
Agreed; I think we should have that template _somewhere_ (TBD) for reference, without making it part of issue templates per se. It should enable the dev team to communicate expectations in a standardised way with QA, without getting in the way of issue submission.